common/aggregation_scheme.py

```diff
@@ -0,0 +1,37 @@
+from utils import constants
+import numpy as np
+
+DEFAULT_AGGREGATION_SCHEME = constants.AGGREGATITON_SCHEME_AVERAGE
+
+
+def get_aggregation_scheme(server_model, CLIENT_IDS, CLIENT_NETWORKS):
+    if DEFAULT_AGGREGATION_SCHEME == constants.AGGREGATITON_SCHEME_AVERAGE:
+        return AggregationAverage(server_model, CLIENT_IDS, CLIENT_NETWORKS)
+
+
+class AggregationAverage():
```

Comment:
i'm not sure if this needs to be a class?

could just define a function that just averages?